src/main/resources/bug_report.md

```diff
@@ -0,0 +1,41 @@
+## Bug description
+Please include steps to reproduce (like `go to...`/`click on...` etc.) + expected and actual behavior.
+
+For more non-trivial issues, we would also appreciate if you included the following details:
+
+### Running environment
+ - Git Machete plugin version - %macheteVersion%
+ - IDE %ide%
+ - Operating system [e.g. Windows]
+
+### IDEA - event logs
```

Comment:
I'd call it `stack trace` (and rename the template param adequately)

Reply:
Although... yeah, theoretically it's called `event logs` by IntelliJ... leaving to your judgement then

Reply:
Actually I prefer stack trace so I will rename it.